sys/net/gnrc/network_layer/ipv6/nib/_nib-arsm.c

```diff
@@ -37,7 +37,7 @@ void _snd_ns(const ipv6_addr_t *tgt, gnrc_netif_t *netif,
     gnrc_pktsnip_t *ext_opt = NULL;
 
 #ifdef MODULE_GNRC_SIXLOWPAN_ND
-    _nib_dr_entry_t *dr = _nib_drl_get_dr();
+    _nib_dr_entry_t *dr = _nib_drl_get(NULL, netif->pid);
```

Comment:
Now this should be after the `assert(netif != NULL)`

Reply:
Done